src/plays/dev-jokes/DevJokes.tsx

```diff
@@ -1,11 +1,17 @@
 import PlayHeader from "common/playlists/PlayHeader";
 import "./devJokes.css";
 import axios from "axios";
-import { useEffect, useState } from "react";
+import {
+  useEffect,
+  useState,
+} from "react";
+import React from "react";
 import Spinner from "./Spinner";
-function DevJokes(props) {
-  const [joke, setjoke] = useState([]);
-  const [spinner, setspinner] = useState(false);
+
+
+function DevJokes(props: any) {
+  const [joke, setjoke] = useState<any[]>([]);
```

Comment:
Can we use proper type for joke?
We should avoid using the `any`.

Reply:
yeah i have tried to do same  i  have created a interface 
code:-
interface jokes{
  punchline: string
  question: string
}

but the problem is the key it throws err
![pr](https://user-images.githubusercontent.com/94097778/193419868-382f70f0-23b4-4bd3-a52d-8392599905f9.png)
or

Reply:
Value is object right and key should be number or string.
Can you try to use string or number for key?

Reply:
I have tried it before it was not working...

Reply:
What code you have tried can you share it?

Reply:
This 

![pr-modufy](https://user-images.githubusercontent.com/94097778/193440159-8b4d3c59-d26d-4e4c-a29c-2c994313d838.png)

Reply:
@yung-coder, did you check what is the structure of API data? does it return the key attribute?

Reply:
It only returns two attributes, `question` and `punchline`.
<img width="1074" alt="image" src="https://user-images.githubusercontent.com/59080942/193440694-106a39ce-c254-4ebb-aa08-8c1536d6ff1b.png">

Reply:
Your interface should only contain these attributes.

Reply:
For the key, you can access the index value from the map callback function like this.

```ts
{joke.map((value,index) => {
                  return (
                    <div key={index}>
                      <h1>{value.question}</h1>
                      <p>{value.punchline}</p>
                    </div>
                  );
                })}
```

Reply:
Thanks @Sachin-chaurasiya got it